python_modules/libraries/dagster-dbt/dagster_dbt_tests/cloud/preview/conftest.py

```diff
@@ -26,6 +33,13 @@
 
 TEST_REST_API_BASE_URL = f"{TEST_ACCESS_URL}/api/v2/accounts/{TEST_ACCOUNT_ID}"
 
+with open(manifest_path) as f:
+    SAMPLE_MANIFEST_JSON = json.load(f)
+
+
+with open(run_results_path) as f:
+    SAMPLE_RUN_RESULTS_JSON = json.load(f)
```

Comment:
nit; let's bury these in a function so that we're not auto loading the file upon module load

Reply:
Could just be a fixture?

Reply:
I went for a function as we mainly need this in conftest